inflammation/models.py

```diff
@@ -32,3 +32,13 @@ def daily_min(data):
     """Calculate the daily min of a 2d inflammation data array."""
     return np.min(data, axis=0)
 
+
+def s_dev(data):
+    """Computes and returns standard deviation for data."""
```

Comment:
The documentation say it computes and returns the standard deviation, but it actually returns a dictionary containing the standard deviation, please specify the output of the function